nixos/modules/services/misc/gitea.nix

```diff
@@ -388,9 +397,11 @@ in
             };
             mailer = {
               ENABLED = true;
-              MAILER_TYPE = "sendmail";
-              FROM = "do-not-reply@example.org";
-              SENDMAIL_PATH = "''${pkgs.system-sendmail}/bin/sendmail";
```

Comment:
This looks like it was already using sendmail? In which case the commit message is quite misleading…